diff --git a/dist/monaco-editor.js b/dist/monaco-editor.js
index 32187d073bfa62031a526f4e20a21ee8ac14c857..e7803203d240a9cd2b765a37a4cd61b2b17e49fc 100644
--- a/dist/monaco-editor.js
+++ b/dist/monaco-editor.js
@@ -178015,7 +178015,7 @@ async function registerProviders(worker, language, getSyncUris, languages) {
 
 function WorkerWrapper$1(options) {
             return new Worker(
-              ""+new URL('assets/editor.worker-C6mZTrQO.js', import.meta.url).href+"",
+              ""+new URL('./assets/editor.worker-C6mZTrQO.js', import.meta.url).href+"",
               {
           type: "module",
           name: options?.name
@@ -178034,7 +178034,7 @@ function getOrCreateModel(uri, lang, value) {
 
 function WorkerWrapper(options) {
   return new Worker(
-    ""+new URL('assets/vue.worker-mfJR090o.js', import.meta.url).href+"",
+    ""+new URL('./assets/vue.worker-mfJR090o.js', import.meta.url).href+"",
     {
       type: "module",
       name: options?.name
